### ISSUE TITLE
Measure tool doesn't match if document contains link

### ISSUE THREAD

**Issue by egli** [sbsdev/dtbook-preptools#4]
The _Measure_ tool of the Prep-tools doesn't work if a text contains links, i.e. if the document contains a `<a>`. It seems to be related to the `<a id="p-..."/>` elements. If you remove those the tool works flawlessly.

``` xml
<?xml version="1.0" encoding="UTF-8"?><!DOCTYPE dtbook [<!ENTITY ndash "&#x2013;"><!ENTITY nbsp "&#x00A0;">]><?oxygen RNGSchema="http://xmlp/schema/dtbook-2005-3-sbs.rng" type="xml"?>
<dtbook xmlns="http://www.daisy.org/z3986/2005/dtbook/"
  xmlns:brl="http://www.daisy.org/z3986/2009/braille/" version="2005-3-sbs-full" xml:lang="de">
  <head>
    <meta name="dc:Title" content="Das halbe Haus"/>
    <meta name="dc:Creator" content="Cynybulk, Gunnar"/>
    <meta name="dc:Publisher"
      content="SBS Schweizerische Bibliothek für Blinde, Seh- und Lesebehinderte"/>
    <meta name="dc:Date" content="2014-08-13"/>
    <meta name="dc:Type" content="Text"/>
    <meta name="dc:Format" content="ANSI/NISO Z39.86-2005"/>
    <meta name="dc:Identifier" content="ch-sbs-883dcdaf-5217-4bd9-9c34-e080a592b14f"/>
    <meta name="dc:Source" content="978-3-8321-9723-0"/>
    <meta name="dc:Language" content="de"/>
    <meta name="dtb:uid" content="ch-sbs-883dcdaf-5217-4bd9-9c34-e080a592b14f"/>
    <meta name="dtb:sourceEdition" content="2014"/>
    <meta name="dtb:sourcePublisher" content="DuMont"/>
    <meta name="dtb:sourceRights" content=""/>
    <meta name="prod:seriesNumber" content=""/>
  </head>
  <book>
    <frontmatter>
      <doctitle>Das halbe Haus</doctitle>
      <docauthor>Gunnar Cynybulk</docauthor>
      <level1 class="titlepage">
        <level2>
          <p>Roman</p>
          <p class="sourcePublisher">Test</p>
        </level2>
      </level1>
    </frontmatter>
    <bodymatter>
      <level1>
        <pagenum page="normal" id="page-7">7</pagenum>
        <a id="p-7"/>
        <p> 12 m 23. ABC A. D. GmbH Sie IV ( être</p>
        <p><a class="pageref" href="#p-7"><brl:emph class="ss">7</brl:emph></a></p>     
      </level1>
    </bodymatter>
  </book>
</dtbook>
```

**Comment by egli** [comment]
As far as I can tell there is a mechanism to skip over certain areas. The original error report mentions that it skips a lot of candidates and jumps right to the line where the `<li>` starts below. Funnily that is right after a `</a>`
``` xml
</brl:select><a class="pageref" href="#p-100"><brl:emph class="ss">100f.</brl:emph></a>).</li>
<li>Erdmandelflocken und Vanille mit 300 ml heißem Wasser mischen. Quellen lassen.</li>

```
Maybe there is indeed a problem with skipping over `<a>` elements that have no children, i.e. `<a id="p-5"/>`.

**Comment by egli** [comment]
Fixed in 70157c0efafce8423e40f63d07f586b7fad209f8
